,py,py-GA-970A-D3,08.05.2018 14:34,file:///home/py/.config/libreoffice/4;